face_model/argface_model/argface_extract_features.py

```
@@ -52,20 +50,20 @@ def extract_features(self, model):
                 try:
                     with torch.no_grad():
                         embedding = model.get_embedding(image_tensor)
+                    self.features.append(embedding.squeeze().numpy())
+                    self.labels.append(label)                    
                 except Exception:
+                    print(f"Error extracting embedding for {image_path}: {e}")
```

Ruff

Undefined name `e`

(F821)